src/tests/Microsoft.VisualStudio.Composition.Tests/AssembliesLazyLoadedTests.cs

```diff
@@ -130,6 +130,31 @@ public async Task ComposableAssembliesLazyLoadedByLazyImport()
             }
         }
 
+        /// <summary>
+        /// Verifies that the assemblies that MEF parts belong to are only loaded when their parts are actually instantiated.
+        /// </summary>
+        [SkippableFact]
```

Comment:
In addition to these tests that you're adding, do tests pass if you merge [`dev/andarno/LazyCompose`](https://github.com/AArnott/vs-mef/commit/d224f51f5253c1fe02dbe69b36b81858370ce743) into your branch? If it passes, please do keep that merge and push it into your branch to lock in the higher test level.

Reply:
They don't pass since there are still some cases where assemblies will get loaded. I'll leave it unmerged for now since I think fully fixing that is out of scope for this PR.